test/FunctionsV2/TestCleanup.cs

```diff
@@ -34,7 +34,9 @@ public TestCleanup(ITestOutputHelper output)
         [Trait("Category", PlatformSpecificHelpers.TestCategory)]
         public async Task CleanupOldAzureStorageTaskHubs()
         {
-            TimeSpan oldTaskHubDeletionThreshold = TimeSpan.FromMinutes(5);
+            // The CI should run at least once a day, so this timespan should avoid deleting
+            // deterministic taskhubs
+            TimeSpan oldTaskHubDeletionThreshold = TimeSpan.FromHours(25);
```

Comment:
I'm a bit confused by these "deterministic taskhubs" we're trying to avoid deleting. Are those taskhubs of currently running tests or something else?

Reply:
Yeah, I ran into an issue where tests would fail because we try to start a taskhub while this test was trying to delete that taskhub.